common/src/main/java/org/opensearch/ml/common/MLModel.java

```diff
@@ -143,6 +143,42 @@ public class MLModel implements ToXContentObject {
     private String connectorId;
     private Guardrails guardrails;
 
+    /**
+     * Model interface is a map that contains the input and output fields of the model, with JSON schema as the value.
+     * Sample model interface:
+     * {
+     *   "interface": {
+     *     "input": {
+     *       "properties": {
+     *         "parameters": {
+     *           "properties": {
+     *             "messages": {
+     *               "type": "string",
+     *               "description": "This is a test description field"
+     *             }
+     *           }
+     *         }
+     *       }
+     *     },
+     *     "output": {
```

Comment:
This just defines output part of remote model , seems not enough, as you can see `"inference_results"` is array, if we just define the inner `output`, user won't know the inference_results structure 
```
{
  "inference_results": [
    {
      "output": [
        {
          "name": "response",
          "dataAsMap": {
            "completion": " There is no single, universally agreed upon meaning of life. The meaning of life is subjective and personal. Some common perspectives include finding happiness, purpose, spiritual fulfillment, connecting with others, contributing value, and leaving a positive legacy. Ultimately, the meaning of life is what you make of it.",
            "stop_reason": "stop_sequence"
          }
        }
      ]
    }
  ]
}
```

Reply:
Changed.